src/main/java/com/google/devtools/build/lib/remote/RemoteSpawnRunner.java

```diff
@@ -124,6 +124,7 @@ public SpawnResult exec(Spawn spawn, SpawnExecutionPolicy policy)
     TreeNode inputRoot = repository.buildFromActionInputs(inputMap);
     repository.computeMerkleDigests(inputRoot);
     Command command = buildCommand(spawn.getArguments(), spawn.getEnvironment());
+    boolean uploadLocalResults = options.remoteUploadLocalResults;
```

Comment:
Can you please elaborate on the use-case for this? It sounds like "I want to leech on the remote cache without ever possibly contributing to it".